pkg/scalers/elasticsearch_scaler.go

```diff
@@ -0,0 +1,276 @@
+package scalers
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/elastic/go-elasticsearch/v7"
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
```

Comment:
Could you please put KEDA related imports to a separate block (indented by one empty line) below the general imports, around line 24. Thanks!